scripts/control_device.py

```diff
@@ -240,15 +240,15 @@
     choices=possible_swing.keys(),
     required=False,
     help=f"thermostat swing, possible values: {possible_swing}",
-    default=None,
+    default="",
```

Comment:
based on the enum this possible values set is constructed from,
an empty string is not a valid option, why do we have it as a default value?

Reply:
because it is required for the control_thermostat function.

The argparse module won't assign value for this argument if its not required and no default value specified

Reply:
see above https://github.com/TomerFi/aioswitcher/pull/492#discussion_r912333644

Reply:
see above